### DISCUSSION TITLE
Help with receiving message in webhook?

### DISCUSSION THREAD

**Discussion by dplusplus1024** [ACINQ/phoenixd#111] [Q&A] [answered]
Hi phoenixd peeps,

It would be super awesome if I could receive the message (memo/note/comment) from the sender in my webhooks. Is this possible?

![image](https://github.com/user-attachments/assets/d3ac9495-e601-4925-ac30-b0caf7ca3e72)

Thanks so much.

**Comment by pm47** [comment]
Hi, that should be doable, will look into it.

**Reply by dplusplus1024** [reply]
Amazing!!

**Reply by pm47** [reply, marked as answer]
It's actually already supported since version 0.3.3

```json
{
  "type": "payment_received",
  "timestamp": 1726581891919,
  "amountSat": 12345,
  "paymentHash": "e4c4a46e3ebb9aff96b2b18c8cb5b4ed2350dd3959ea579f36724ebed43d7e69",
  "externalId": null,
  "payerNote": "Hello there! 👋",
  "payerKey": "02a253612063909c59bfa766d3c459a3eb3ec50c51f938099397158200d025f213"
}
```

**Reply by dplusplus1024** [reply]
![image](https://github.com/user-attachments/assets/3027974f-4a17-4961-abc5-ba348aac74fd)
